### DISCUSSION TITLE
Private Key of Historical certificate

### DISCUSSION THREAD

**Discussion by afmorton** [Venafi/venafips#296] [Q&A]
You can easily obtain the CER version of a historical certificate using  Get-VdcCertificate -ID $CertPath -IncludePreviousVersions then take  each VaultID pass to Export-VdcVaultObject -ID <VaultID> -OutPath 'c:\cert-export', which ends up with all the CER files in the supplied path.

However is there a way to export the associated Private Key (eg. PKCS #12) as I have yet to find a combination that would achieve this ?

**Comment by gdbarron** [comment]
Is the historical certificate using the same private key as the current certificate?  This is typically the case and if so you can use `Export-VdcCertificate -Path $CertPath -pkcs8 -PrivateKeyPassword 'mySecretPassword1!' -OutPath $out` to create the key file.  If you want a p12 you'll need to convert the cer and key.

**Reply by afmorton** [reply]
Sadly they are different Private keys.
